pkg/ingester/metrics_test.go

```diff
@@ -12,11 +12,11 @@ import (
 func TestShipperMetrics(t *testing.T) {
 	mainReg := prometheus.NewRegistry()
 
-	shipper := newShipperMetrics(mainReg)
+	tsdbMetrics := newTsdbMetrics(mainReg)
 
-	populateShipperMetrics(shipper.newRegistryForUser("user1"), 12345)
-	populateShipperMetrics(shipper.newRegistryForUser("user2"), 85787)
-	populateShipperMetrics(shipper.newRegistryForUser("user3"), 999)
+	tsdbMetrics.setRegistryForUser("user1", populateShipperMetrics(12345))
```

Comment:
Could you add 1 of the two new per-user metrics to the `populateShipperMetrics()` so that we test it too, please?